scene/3d/visual_instance.cpp

```diff
@@ -108,6 +108,17 @@ void VisualInstance::_notification(int p_what) {
 			if (_is_vi_visible() && is_physics_interpolated()) {
 				VisualServer::get_singleton()->instance_reset_physics_interpolation(instance);
 			}
+#if defined(DEBUG_ENABLED) && defined(TOOLS_ENABLED)
+			else if (GLOBAL_GET("debug/settings/physics_interpolation/enable_warnings")) {
+				String node_name = is_inside_tree() ? String(get_path()) : String(get_name());
+				if (!_is_vi_visible()) {
+					WARN_PRINT("[Physics interpolation] NOTIFICATION_RESET_PHYSICS_INTERPOLATION only works with unhidden nodes:  \"" + node_name + "\".");
```

Comment:
Should it be `WARN_PRINT_ONCE` to avoid spamming?

Reply:
Also maybe `visible` is clearer than `unhidden`?

Reply:
In this case, `WARN_PRINT` should be fine I think, because `reset` is rarely called, and we want a warning for each node that calls it incorrectly. But the spaces is a typo I'll change it. :+1: